drone.sh

```diff
@@ -100,10 +100,15 @@ fi
 echo "$SSH_KEY" > $keyfile
 chmod 0600 $keyfile
 
-# Parse SSH commands
+# Parse SSH precommands
 function join_with { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
-IFS=','; read -ra COMMANDS <<< "$PLUGIN_SCRIPT"
-script=$(join_with ' && ' "${COMMANDS[@]}")
+IFS=','; read -ra COMMANDS <<< "$PLUGIN_PRESCRIPT"
+prescript=$(join_with ' && ' "${COMMANDS[@]}")
+
+# Parse SSH postcommands
+function join_with { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
```

Comment:
You don't have to define this function twice if you want to use it twice...

Reply:
removed duplicate function.